opentelemetry-auto-instrumentation/src/opentelemetry/auto_instrumentation/auto_instrumentation.py

```diff
@@ -15,22 +15,22 @@
 # limitations under the License.
 
 from logging import getLogger
-from runpy import run_path
+from os import environ, execl
+from os.path import abspath, dirname, pathsep
+from shutil import which
 from sys import argv
 
-from pkg_resources import iter_entry_points
-
 logger = getLogger(__file__)
 
 
 def run() -> None:
 
-    for entry_point in iter_entry_points("opentelemetry_instrumentor"):
-        try:
-            entry_point.load()().instrument()  # type: ignore
-            logger.debug("Instrumented %s", entry_point.name)
+    python_path = environ.get("PYTHONPATH", "")
+    filedir_path = dirname(abspath(__file__))
+
+    if filedir_path not in python_path:
+        environ["PYTHONPATH"] = pathsep.join([filedir_path, python_path])
```

Comment:
Shouldn't this logic guarantee that `filedir_path` is always the first element on  `PYTHONPATH`?

Reply:
Fixed